liblangutil/Scanner.cpp

```diff
@@ -389,9 +394,11 @@ Token Scanner::scanMultiLineDocComment()
 			}
 			else if (!m_source->isPastEndOfInput(1) && m_source->get(0) == '*' && m_source->get(1) != '/')
 			{ // skip first '*' in subsequent lines
+				m_char = m_source->advanceAndGet(1);
```

Comment:
`m_source->advanceAndGet(1)` instead of `m_source->advanceAndGet(2)` becuase the older one assumes that `*` has to be followed by a space. This cause two problems:
1. Reading past `\n` if `*` is followed by just EOL.
2. Problem with parsing `*@notice`.

Doing so also makes singleline and multiline docstringcomments consistent.